diff --git a/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt b/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
index fc56526..08fa36b 100644
--- a/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
+++ b/node_modules/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt
@@ -132,6 +132,10 @@ class ExpoCameraView(

   private var previewView = PreviewView(context).apply {
     elevation = 0f
+    
+    
+    
+    implementationMode = PreviewView.ImplementationMode.COMPATIBLE
   }
   private val scope = CoroutineScope(Dispatchers.Main)
   private var shouldCreateCamera = false
